Python/Product/PythonTools/PythonTools/Debugger/DebugPyVersionHelper.cs

```diff
@@ -25,81 +25,81 @@
+        [JsonProperty("debugpy")]
+        public DebugPyDebuggerVersion Debugger { get; set; }
 
         [JsonProperty("python")]
         public PythonVersionInfo Python { get; set; }
 
         [JsonProperty("platform")]
-        public PtvsdPlatfromInfo Platfrom { get; set; }
+        public DebugPyPlatfromInfo Platfrom { get; set; }
 
         [JsonProperty("process")]
-        public PtvsdProcessInfo Process { get; set; }
+        public DebugPyProcessInfo Process { get; set; }
 
     }
 
-    internal class PtvsdVersionRequest : DebugRequestWithResponse<PtvsdVersionArguments, PtvsdVersionResponse> {
-        public PtvsdVersionRequest(): base("ptvsd_systemInfo") {
+    internal class DebugPyVersionRequest : DebugRequestWithResponse<DebugPyVersionArguments, DebugPyVersionResponse> {
+        public DebugPyVersionRequest(): base("debugpySystemInfo") {
         }
     }
 
-    internal class PtvsdVersionHelper {
-        public static void VerifyPtvsdVersion(PtvsdVersionArguments args, PtvsdVersionResponse response) {
+    internal class DebugPyVersionHelper {
+        public static void VerifyDebugPyVersion(DebugPyVersionArguments args, DebugPyVersionResponse response) {
             if (PackageVersion.TryParse(response.Debugger.Version, out PackageVersion runningVersion)) {
-                var bundledPtvsdVersion = PackageVersion.Parse(PtvsdVersion.Version);
-                if (runningVersion.CompareTo(bundledPtvsdVersion) < 0) {
-                    ShowPtvsdMessage(
+                var bundledDebugPyVersion = PackageVersion.Parse(DebugPyVersion.Version);
+                if (runningVersion.CompareTo(bundledDebugPyVersion) < 0) {
+                    ShowDebuggingErrorMessage(
                         Strings.InstalledPtvsdOutdatedTitle,
-                        Strings.InstalledPtvsdOutdatedMessage.FormatUI(response.Debugger.Version, PtvsdVersion.Version),
+                        Strings.InstalledPtvsdOutdatedMessage.FormatUI(response.Debugger.Version, DebugPyVersion.Version),
                         allowDisable: false,
                         isError: false
                     );
                 }
             }
         }
 
-        public static void VerifyPtvsdVersionError(PtvsdVersionArguments args, ProtocolException ex) {
-            ShowPtvsdMessage(
+        public static void ShowDebugPyVersionError(DebugPyVersionArguments args, ProtocolException ex) {
+            ShowDebuggingErrorMessage(
                 Strings.InstalledPtvsdOutdatedTitle,
-                Strings.InstalledPtvsdOutdatedMessage.FormatUI("unknown", PtvsdVersion.Version),
+                Strings.InstalledPtvsdOutdatedMessage.FormatUI("unknown", DebugPyVersion.Version),
                 allowDisable: false,
                 isError: false
             );
         }
 
-        public static void VerifyPtvsdVersionLegacy() {
-            ShowPtvsdMessage(
+        public static void ShowLegacyPtvsdVersionError() {
```

Comment:
This will confusing if this gets triggered.

You have an old version of ptvsd installed in the current environment (version {0}), but version {1} is included with Visual Studio. Please upgrade or uninstall ptvsd to get the latest functionality.

where {0} is 3.* and {1} is 1.0

Reply:
my guess is that it won't be possible to see it though, as we can't accidentally launch legacy ptvsd when we import debugpy.